tracing-subscriber/src/fmt/format/json.rs

```diff
@@ -22,13 +22,25 @@ use tracing_log::NormalizeEvent;
 /// Marker for `Format` that indicates that the verbose json log format should be used.
 ///
 /// The full format includes fields from all entered spans.
-#[derive(Default, Debug, Copy, Clone, Eq, PartialEq)]
+///
+/// # Example Output
+///
+/// ```ignore,json
+/// {"timestamp":"Feb 20 11:28:15.096","level":"INFO","target":"mycrate","fields":{"message":"some message", "key": "value"}}
+/// ```
+///
+/// # Options
+///
+/// - `Json::flatten_event` can be used to enable flattening event metadata into the root
+/// object.
```

Comment:
Since the main way users _interact_ with the JSON formatter is via the `LayerBuilder::json` method, it might be good to have this there as well, or make sure that method links here?